index.js

```diff
@@ -25,6 +25,7 @@ module.exports = {
       throw new SilentError('Cannot find app blueprint for generating test-app!');
     }
 
+    let addonInfo = addonInfoFromOptions(options);
```

Comment:
I found that around here I wanted to add
```
appBlueprint.addPackagesToProject.call(appBlueprint, [{ '@babel/core': 'latest' }]);
```

but ran in to the same problem with ember-cli-typescript.


this `@babel/core` issue needs to be fixed upstream before we can add `@babel/core` support.
(I think that means adding `@babel/core` to the ember-cli app blueprint?)